### PR TITLE
OBSDOCS-1302: Logging Z-Stream Release Notes - 5.8.13

### PR DESCRIPTION
**Pull request by smunje1**
Change type: Doc update; Logging Z-Stream Release Notes - 5.8.13
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1302

Fix Version: 4.12, 4.13, 4.14, 4.15

Doc Preview: https://82689--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging_release_notes/logging-5-8-release-notes.html#cluster-logging-release-notes-5-8-13_logging-5-8-release-notes

SME review completed: @periklis 
QE review completed: @anpingli 
Peer review completed: @mburke5678

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

**Comment by anpingli** [comment]
LGTM

File: modules/logging-release-notes-5-8-13.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// module included in logging-5-8-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-8-13_{context}"]
+= Logging 5.8.13
+This release includes link:https://access.redhat.com/errata/RHSA-2024:7237[OpenShift Logging Bug Fix Release 5.8.13] and link:https://access.redhat.com/errata/RHBA-2024:7230[OpenShift Logging Bug Fix Release 5.8.13].
+
+[id="openshift-logging-5-8-13-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter was not applied to the Fluentd configuration when Fluentd was used as the collector type, causing HTTP timeouts to be misconfigured. With this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter is now correctly applied, ensuring Fluentd honors the specified timeout and handles HTTP connections according to the user’s configuration. (link:https://issues.redhat.com/browse/LOG-5210[LOG-5210])
```
Comment (mburke5678) [inline modules/logging-release-notes-5-8-13.adoc:10]:
```suggestion
* Before this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter was not applied to the Fluentd configuration when Fluentd was used as the collector type, causing HTTP timeouts to be misconfigured. With this update, the `clusterlogforwarder.spec.outputs.http.timeout` parameter is now correctly applied, ensuring that Fluentd honors the specified timeout and handles HTTP connections according to the user’s configuration. (link:https://issues.redhat.com/browse/LOG-5210[LOG-5210])
```

File: modules/logging-release-notes-5-8-13.adoc [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+* Before this update, the Elasticsearch Operator did not issue an alert to inform users about the upcoming removal, leaving existing installations unsupported without notice. With this update, the Elasticsearch Operator will trigger a continuous alert on OCP 4.16 and later versions, notifying users of its removal from the catalog in November 2025. (link:https://issues.redhat.com/browse/LOG-5966[LOG-5966])
```
Comment (mburke5678) [inline modules/logging-release-notes-5-8-13.adoc:12]:
```suggestion
* Before this update, the Elasticsearch Operator did not issue an alert to inform users about the upcoming removal, leaving existing installations unsupported without notice. With this update, the Elasticsearch Operator will trigger a continuous alert on {product-title} version 4.16 and later, notifying users of its removal from the catalog in November 2025. (link:https://issues.redhat.com/browse/LOG-5966[LOG-5966])
```

File: modules/logging-release-notes-5-8-13.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+* Before this update, the Red Hat OpenShift Logging Operator was unavailable on OCP 4.16 and later catalogs, preventing Telco customers from completing their certifications for the upcoming Logging 6.0 release. With this update, the Red Hat OpenShift Logging Operator is now available on OCP 4.16 and OCP 4.17, resolveing the issue. (link:https://issues.redhat.com/browse/LOG-6103[LOG-6103])
```
Comment (mburke5678) [inline modules/logging-release-notes-5-8-13.adoc:14]:
```suggestion
* Before this update, the Red Hat OpenShift Logging Operator was unavailable on {product-title} version 4.16 and later, preventing Telco customers from completing their certifications for the upcoming Logging 6.0 release. With this update, the Red Hat OpenShift Logging Operator is now available on {product-title} versions 4.16 and 4.17, resolveing the issue. (link:https://issues.redhat.com/browse/LOG-6103[LOG-6103])
```

File: modules/logging-release-notes-5-8-13.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+* Before this update, the Elasticsearch Operator was not available in the OCP 4.17 and 4.18 catologs, preventing installations for ServiceMesh, Kiali, and Distributed Tracing. With this update, the Elasticsearch Operator properties have been expanded for OCP versions 4.17 and 4.18, resolving the issue and allowing ServiceMesh, Kiali, and Distributed Tracing operators to install their stacks. (link:https://issues.redhat.com/browse/LOG-6134[LOG-6134])
```
Comment (mburke5678) [inline modules/logging-release-notes-5-8-13.adoc:16]:
```suggestion
* Before this update, the Elasticsearch Operator was not available in the {product-title} versions 4.17 and 4.18, preventing the installation of ServiceMesh, Kiali, and Distributed Tracing. With this update, the Elasticsearch Operator properties have been expanded for {product-title} versions 4.17 and 4.18, resolving the issue and allowing ServiceMesh, Kiali, and Distributed Tracing operators to install their stacks. (link:https://issues.redhat.com/browse/LOG-6134[LOG-6134])
```

**Comment by mburke5678** [comment]
@smunje1 A couple of comments. Otherwise LGTM

**Comment by smunje1** [comment]
Hello @mburke5678 , I have incorporated your suggestions and proceeding for merge review. Thank you!

**Pull request merged**